src/TypeSpec.Extension/Emitter.Csharp/src/type/usage.ts

```diff
@@ -5,5 +5,6 @@ export enum Usage {
     None = "None",
     Input = "Input",
     Output = "Output",
-    RoundTrip = "RoundTrip"
+    RoundTrip = "RoundTrip",
+    Multipart = "Multipart"
```

Comment:
I'm not sure if it's the right `usage`

Reply:
It's actually a media type

Reply:
well because this kind of media type is contagious on models, Michael suggests that we put it on Usage as a flag of that model.